File: Aplikacja/Serwer/CsharpServerApp/oto-auto-c-sharp-server/oto-auto-c-sharp-server/Logic/Offers/Models/CreateOffer/CreateOfferRequest.cs
using oto_auto_c_sharp_server.Logic.Offers.Models.CreateOffer.Equipments;

namespace oto_auto_c_sharp_server.Logic.Offers.Models.CreateOffer;

public class CreateOfferRequest
{
    public AdditionalTechnicalDataForm? AdditionalTechnicalDataForm { get; set; }
    public BasicInfo? BasicInfo { get; set; }
    public DealerDataForm? DealerDataForm { get; set; }
    public EquipmentTypeForm? EquipmentTypeForm { get; set; }
    public MainFeatures? MainFeatures { get; set; }
    public IEnumerable<OfferImages>? OfferImages { get; set; }
    public PriceDataForm? PriceDataForm { get; set; }
    public TechnicalDataForm? TechnicalDataForm { get; set; }
    public VehicleDescription? VehicleDescription { get; set; }
    public VehicleType? VehicleType { get; set; }
}

File: Aplikacja/Mobile/OtoAuto/app/src/main/java/org/masters_thesis/otoauto/model/createOffer/CreateOfferRequest.kt
package org.masters_thesis.otoauto.model.createOffer

data class CreateOfferRequest (
    val additionalTechnicalDataForm: AdditionalTechnicalDataForm?,
    val basicInfo: BasicInfo?,
    val dealerDataForm: DealerDataForm?,
    val equipmentTypeForm: EquipmentTypeForm?,
    val mainFeatures: MainFeatures?,
    val offerImages: List<OfferImages>?,
    val priceDataForm: PriceDataForm?,
    val technicalDataForm: TechnicalDataForm?,
    val vehicleDescription: VehicleDescription?,
    val vehicleType: VehicleType?
)